File: VB/WindowsApplication1/MyColorEdit/MyPopupColorEditForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Popup

Namespace CustomEditors
	Public Class MyPopupColorEditForm
		Inherits PopupColorEditForm
		Public Sub New(ByVal ownerEdit As ColorEdit)
			MyBase.New(ownerEdit)
		End Sub
		Protected Overrides Function CreateColorListBox() As ColorListBox
			Return New MyColorListBox()
		End Function
	End Class
End Namespace

File: CS/WindowsApplication1/MyColorEdit/MyPopupColorEditForm.cs
using System;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Popup;

namespace CustomEditors
{
    public class MyPopupColorEditForm : PopupColorEditForm
    {
        public MyPopupColorEditForm(ColorEdit ownerEdit) : base(ownerEdit) { }
        protected override ColorListBox CreateColorListBox()
        {
            return new MyColorListBox();
        }
    }
}
